pages/public_cloud/containers_orchestration/managed_kubernetes/migrate-loadbalancer-iolb-to-octavia/guide.en-gb.md

```diff
@@ -0,0 +1,107 @@
+
+> [!warning]
+>
+> As Loadbalancer for Kubernetes and Public Cloud LoadBalancer do not use the same solution for Public IP allocation, it is not possible to keep the existing public IP of your Loadbalancer for Kubernetes. Changing the LoadBalancer class of your Service will lead to the creation of a new Loadbalancer and the allocation of a new Public IP (Floating IP).
+
+## Migration of your LoadBalancer
+
+> [!warning]
+>  
+> Starting with MKS version **1.32**, any upgrade attempt will be blocked if an **IOLB** LoadBalancer service is still present in the cluster.  
+>  
+> Action required**: Before upgrading, you must **migrate your services to the Public Cloud LoadBalancer (Octavia)** by following the steps described in this guide.  
+>  
+> If you attempt to upgrade without first migrating, an error will be returned, preventing the upgrade.
+
+In order to migrate from an existing [LoadBalancer for Kubernetes](/links/public-cloud/load-balancer-kubernetes) to a [Public Cloud LoadBalancer](/links/public-cloud/load-balancer) you will have to modify an existing Service and change its LoadBalancer class.
+
+Your existing LoadBalancer Service using [LoadBalancer for Kubernetes](/links/public-cloud/load-balancer-kubernetes) should have the following annotation:
+
+```yaml
+annotations:
+  loadbalancer.ovhcloud.com/class: "iolb"
+```
+
+##### Step 1 - Edit your Service to change the LoadBalancer class to 'octavia'
+
+```yaml
+annotations:
+  loadbalancer.ovhcloud.com/class: "octavia" // not required for clusters running kubernetes versions >= 1.31, you can just remove the annotation.
+```
+
+##### Step 2 - Apply the change
+
+```yaml
+kubectl apply -f your-service-manifest.yaml
+```
+
+## Perform a DNS switch
+
+### Check the current TTL
+
+By default, DNS servers cache the IP address of a domain for a period defined by the TTL (Time-To-Live).
+
+A TTL that is too long can slow down the transition by forcing some users to wait several hours before accessing the new Load Balancer. To avoid this, we recommend temporarily reducing the TTL before updating the IP.
+
+Before changing the TTL, it is important to know its current value. To do this, run the following command in a terminal:
+
+```bash
+dig yourdomain.com +noall +answer
+
+yourdomain.com.             224     IN      A       17*.***.**.*4
+```
+
+Here, 224 corresponds to the TTL in seconds (i.e. approximately 4 minutes). This means that the DNS servers keep this IP in cache for this length of time before checking for an update.
+
+### Lower the TTL before migration
+
+To lower the TTL, follow these steps:
+
+- Access your DNS management console.
+- Find the A record (or CNAME) associated with your domain.
+- Change the TTL of your A or CNAME record by reducing it to 300 seconds (5 min).
```

Comment:
```suggestion
- Change the TTL of your A or CNAME record by reducing it to 300 seconds (5 min).
```

Given the above exemple you suggest a change for "approximately 4 minutes" to 5 minutes